chore: Rename sockets to wocket

ebebbington opened the pull request:
## Summary

* Rename sockets to wocket

ebebbington commented:
@crookse might need your help - unsure what is cuasing these errors:

![image](https://user-images.githubusercontent.com/47337480/98467928-fa868980-21cf-11eb-8694-c3fb8bfe0836.png)

I've checked the `app.vue` and `_app.js` but couldnt see anything that might be the cause

crookse commented:
i think that's my bad. take a look at `app_root.vue` line 25. i set `required: ""`. it should be `default: ""`. because later down in the file, that prop gets `.split()` on commas (line 41).

also, that `.split()` on the `news_tags` prop should probably go in the `if` block below it

ebebbington commented:
oh wow that fixed it lol, i removed that line for the split on the news tag as it was never used

ebebbington commented:
Removed example code props from some drash vue components as they were producing errors (we no longer have example code dirs)

Also saw errors in wocket where some components were nott registered - i remember eric saying he thought we didn't have to import them into the vue files.. turns out we don't, but a line needs to be added to the `_app.js` file, which I have added for the other modules, so from now on, importing components just to use them is not needed, we can just 'use the component'

ebebbington commented:
Ready for review
